website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -43,6 +43,9 @@ npm --save install react react-dom react-intl
 * [TabularMainPage](#tabularmainpage)
 * [TabularDetailPage](#tabulardetailpage)
 * [TabHeader](#tabheader)
+* [CustomFormMainPage](#customformmainpage)
+* [FormMainPage](#formmainpage)
+* [InfoMainPage](#infomainpage)
```

Comment:
The components appear in the reverse order both in content and in sidebar so perhaps it would be better to adjust it here:
```suggestion
* [InfoMainPage](#infomainpage)
* [FormMainPage](#formmainpage)
* [CustomFormMainPage](#customformmainpage)
```

Reply:
Updated [6417510](https://github.com/commercetools/merchant-center-application-kit/pull/2745/commits/641751086c14b5f72b8472b771eb34065bfa415e)